pkg/tenant/api/v1/api.go

```diff
@@ -75,6 +78,29 @@ type StatusReply struct {
 // Tenant Requests and Responses
 //===========================================================================
 
+type RegisterRequest struct {
+	Name     string `json:"name"`
+	Email    string `json:"email"`
+	Password string `json:"password"`
+	PwCheck  string `json:"pwcheck"`
+}
+
+type RegisterReply struct {
+	Email   string `json:"email"`
+	Message string `json:"message"`
+	Role    string `json:"role"`
+}
```

Comment:
Just an FYI that we will have to update this after tomorrow's feature session - I'm aware that we aren't collecting all of the registration data but needed to catch up with the front end team to find out what data was coming in.

When we create the story to update the register requests structs it should update for _both_ tenant and Quarterdeck.